senders/discord/init.go

```diff
@@ -16,6 +17,12 @@ const (
 	workerName      = "DiscordBot"
 )
 
+// Structure that represents the Discord configuration in the YAML file
+type Discord struct {
```

Comment:
We can make this type (and other sender configuration types) private, since it is only used in this package

Reply:
fixed